### ISSUE TITLE
Deprecation of discovery_info['ssdp_location']

### ISSUE THREAD

**Issue by remyvv** [tmjo/heos_custom#6]
Since version 2021.12 of Home Assistant, this warning pops up in the logs:

```
WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that accessed discovery_info['ssdp_location'] instead of discovery_info.ssdp_location; this will fail in version 2022.6. Please report issue to the custom component author for heos using this method at custom_components/heos/config_flow.py, line 29: hostname = urlparse(discovery_info[ssdp.ATTR_SSDP_LOCATION]).hostname
```

If you'd like i can try to submit a PR fixing this deprecation?

**Comment by tmjo** [comment]
Hi, thanks for your feedback! I didn't update yet, so didn't see it. If you'd like to send a PR, it would be appreciated. Also, I think there may be official support for HEOS grouping as of this release, but didn't confirm it myself yet. See [here](https://community.home-assistant.io/t/heos-group-support/126501/98) in last couple of posts for info.

**Comment by remyvv** [comment]
You're right. The grouping feature was the reason for me to use this custom component.
I have now switched over to the bundled integration, and have confirmed it works.

If you want this issue can be closed if there is no intention to further develop this custom component.

**Comment by tmjo** [comment]
Sounds good, I did some basic testing with it too and it looks good. I guess I will deprecate this custom integration unless I decide to add some other features, for instance the 'media browser' stuff is not yet in the official integration. I started on that too some time back, but never completed it. I keep this open just to remind to fix the warning anyways.
